scan/lib/scan/test_command_generator.rb

```diff
@@ -45,6 +45,7 @@ def options # rubocop:disable Metrics/PerceivedComplexity
       options << "-enableCodeCoverage #{config[:code_coverage] ? 'YES' : 'NO'}" unless config[:code_coverage].nil?
       options << "-enableAddressSanitizer #{config[:address_sanitizer] ? 'YES' : 'NO'}" unless config[:address_sanitizer].nil?
       options << "-enableThreadSanitizer #{config[:thread_sanitizer] ? 'YES' : 'NO'}" unless config[:thread_sanitizer].nil?
+      options << "-testPlan #{config[:testplan]}" if config[:testplan]
```

Comment:
Does this work with Xcode 10, or we don't care about older versions?

Reply:
IMO, this should be wrapped in the `if FastlaneCore::Helper.xcode_at_least?(11)` block.

Reply:
Makes sense!